models/pattern/core/pattern.go

```diff
@@ -323,14 +323,27 @@ func processCytoElementsWithPattern(eles []cytoscapejs.Element, pf *Pattern, cal
 	}
 	return nil
 }
+func manifestIsEmpty(manifests []string) bool {
+	for _, m := range manifests {
+		x := strings.TrimSpace(strings.Trim(m, "\n"))
+		if x != "---" && x != "" {
```

Comment:
Won't this match on long manifests that have multiple `---` separators?

Reply:
This logic essentially says - After separating by "\n---\n", if we find some other characters except for "---" - we mark that as a non-empty manifest